src/main/java/io/anserini/rerank/lib/ScoreTiesAdjusterReranker.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Anserini: An information retrieval toolkit built on Lucene
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.rerank.lib;
+
+import io.anserini.rerank.Reranker;
+import io.anserini.rerank.RerankerContext;
+import io.anserini.rerank.ScoredDocuments;
+
+public class ScoreTiesAdjusterReranker implements Reranker {
+  @Override
+  public ScoredDocuments rerank(ScoredDocuments docs, RerankerContext context) {
+    int dup = 0;
+    for (int i=0; i<docs.documents.length; i++) {
+      docs.scores[i] = Math.round(docs.scores[i] * 1e4f) / 1e4f;
+
+      // If we encounter ties, we want to perturb the final score a tiny bit.
```

Comment:
nit: can we make these comments shorter per line? so that I can view them on my phone without swipe left and right?

Reply:
I'm going by IntelliJ default, which is 120 characters... how much narrower do you want to make it?

Reply:
Mine is 100?